src/docs/usage.rst

```diff
@@ -1,4 +1,57 @@
-Usage
-=====
+Getting Started: Haplotype quantification
+======================================================
 
-Describe the usage.
\ No newline at end of file
+HLA quantification
+********
+
+Preparation
+-----------------
+
+HLA quantification is carried out in 3 steps. Before starting, some files have to be downloaded. First, HLA alleles and the associated XML file has to be downloaded from the IPD-IMGT/HLA database using::
+
+  wget ftp://ftp.ebi.ac.uk/pub/databases/ipd/imgt/hla/hla_gen.fasta
+  wget ftp://ftp.ebi.ac.uk/pub/databases/ipd/imgt/hla/xml/hla.xml.zip
+
+Then, the xml file has to be unzipped using::
+
+  unzip hla.xml.zip
+
+Second, allele frequency table is downloaded using R. For that to happen, open a terminal and type::
+
+  R
+
+and paste the following lines::
+
+  download.file("https://github.com/Genentech/midasHLA/blob/11bde30cbbf11b34f2dea29a6284371a9c1e9440/data/allele_frequencies.rda?raw=true", "allele_frequencies.csv")
```

Comment:
Why using R here? Can't you just use wget here as well?

Reply:
Not really, the repository only offers the R Data File

Reply:
Ah, and this function call converts it into a csv file? I guess it is also possible to call this as a bash one-liner instead of opening an interactive R session

Reply:
yes, I fixed it now